### ISSUE TITLE
Can't make latest version

### ISSUE THREAD

**Issue by ekneale** [SuperK-Soft/SKAnalysis#1]
error: 'class Store' has no member named 'GetMap'
error: no matching function for call to 'BStore::Set(const char [19], unsigned int)'
error: 'class Store' has no member named 'Erase'

**Comment by marc1uk** [comment]
This is a dependency on some required modifications to ToolFrameworkCore that are still outstanding
 (see https://github.com/ToolFramework/ToolFrameworkCore/pull/1)
For the time being please use the following branch as a workaround until this gets merged in.
```
cd Dependencies/ToolFrameworkCore
git remote add marc1uk https://github.com/:marc1uk/ToolFrameworkCore.git
git fetch marc1uk
git checkout mods
make clean && make
```

**Comment by JackFannon** [comment]
I think there's a typo here. It should be:
``` git remote add marc1uk git@github.com:marc1uk/ToolFrameworkCore.git ```
instead of
```git remote add marc1uk https://github.com/:marc1uk/ToolFrameworkCore.git```

**Comment by marc1uk** [comment]
yeah the `git@github.com` version can be used, or the specific typo in the `https` version is the `:` after `github.com/`:
```
https://github.com/marc1uk/ToolFrameworkCore.git
```
is fine.
